cql3/statements/schema_altering_statement.cc

```diff
@@ -64,7 +64,7 @@ schema_altering_statement::execute0(query_processor& qp, service::query_state& s
     auto& mm = qp.get_migration_manager();
     ::shared_ptr<cql_transport::event::schema_change> ce;
 
-    if (mm.is_raft_enabled() && this_shard_id() != 0) {
+    if (this_shard_id() != 0) {
```

Comment:
The helper itself stays anyway, so what's the point in removing it from here now?

Reply:
I left the only use of `is_raft_enabled` which actually "does something" (performs schema changes using group 0 if raft is enabled), and I removed all uses which are unnecessary. This is so the upgrade PR can focus on solving the upgrade problem and not fixing "leftovers" that could have been fixed earlier. Hence I moved the "fixing leftovers" part to this separate PR.

The upgrade PR removes the final use of the helper and the helper itself.